content/en/docs/v3.6/tech-overview/design-client.md

```diff
@@ -138,3 +132,5 @@ Client-side keepalive ping still does not reason about network partitions. Strea
 ![client-balancer-figure-07.png](../img/client-balancer-figure-07.png)
 
 Currently, retry logic is handled manually as an interceptor. This may be simplified via [official gRPC retries](https://github.com/grpc/proposal/blob/master/A6-client-retries.md).
+
+*Authors: Gyuho Lee (github.com/gyuho, Amazon Web Services, Inc.), Joe Betz (github.com/jpbetz, Google Inc.)*
```

Comment:
I wonder if we still need author attribution here? This isn't a blog post, and I think that the commit history would show the file authorship.

Reply:
Removed authorship lines.